optuna/integration/sklearn.py

```diff
@@ -511,21 +511,24 @@ class OptunaSearchCV(BaseEstimator):
             Actual study.
 
     Examples:
-        >>> import optuna
-        >>> from sklearn.datasets import load_iris
-        >>> from sklearn.svm import SVC
-        >>> clf = SVC(gamma='auto')
-        >>> param_distributions = {
-        ...     'C': optuna.distributions.LogUniformDistribution(1e-10, 1e+10)
-        ... }
-        >>> optuna_search = optuna.integration.OptunaSearchCV(
-        ...     clf,
-        ...     param_distributions
-        ... )
-        >>> X, y = load_iris(return_X_y=True)
-        >>> optuna_search.fit(X, y) # doctest: +ELLIPSIS
-        OptunaSearchCV(...)
-        >>> y_pred = optuna_search.predict(X)
+
+        .. testcode::
+
+                import optuna
+                from sklearn.datasets import load_iris
+                from sklearn.svm import SVC
+
+                clf = SVC(gamma='auto')
+                param_distributions = {
+                    'C': optuna.distributions.LogUniformDistribution(1e-10, 1e+10)
+                }
+                optuna_search = optuna.integration.OptunaSearchCV(
+                    clf,
+                    param_distributions
+                )
+                X, y = load_iris(return_X_y=True)
+                optuna_search.fit(X, y) # doctest: +ELLIPSIS
```

Comment:
Could you please tell me why `+ELLIPSIS` is needed here?

Reply:
Thanks.
This `+ELLIPSIS` option comes from original code for doctest.
I think we can delete it because this option does not make sense any more in our test.
i.e. We don't check output of `optuna_search.fit` in this test.

Reply:
It makes sense, thank you for clarification. :+1: